uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/components/staff-leave-report.html

```diff
@@ -384,14 +387,17 @@
                                 <i class="fa fa-ellipsis-v"></i>
                               </a>
                               <ul class="dropdown-menu dropdown-menu-right" uib-dropdown-menu>
-                                <li ng-repeat="action in report.actionsFor(request)">
-                                  <a ng-if="action === 'cancel' && report.canCancel(request)" href
-                                    ng-click="report.action(request, action)">
+                                <li ng-repeat="action in ::report.actionsFor(request)"
+                                  ng-switch="['edit', 'respond', 'view'].indexOf(action) > -1">
+                                  <a href
+                                    ng-switch-when="true"
```

Comment:
why? `ng-switch` seems a perfectly elegant solution to me in this case (and i think we use it in manager reports as well)